README.md

```diff
@@ -1,41 +1,34 @@
-# [WIP] skupper [WIP]
+# Skupper
 
-Note: this repository is a work in progress targeted for the consolidation
-and refactoring of the skupper implementation.
+Skupper enables cloud communication by enabling you to create a Virtual Application Network.
 
-Command line tool for setting up and managing skupper installations
+This application layer network decouples addressing from the underlying network infrastructure.
+This enables communication without a VPN.
 
-## Usage
+You can use Skupper to create a network from namespaces in one or more Kubernetes clusters as described in the [Getting Started](https://skupper.io/start/index.html).
+This guide describes a simple network, however there are no restrictions on the topology created which can include redundant paths.
 
-See `skupper help` or `skupper <command> --help` for details.
+Connecting one Skupper site to another site enables communication both ways.
+Communication can occur using any path available on the network, that is, direct connections are not required to enable communication.
```

Comment:
@grs  Another way to say this would be mention that a topology is a directed graph, would that help?

Reply:
Yes, that is better.